website/client/src/components/achievements/chooseClass.vue

```diff
@@ -121,44 +121,48 @@
   }
 
   .class-explanation {
-    font-size: 16px;
-    margin: 1.5em auto;
+    font-size: 1rem;
+    margin: 24px auto;
   }
 
   #classOptOutBtn {
     cursor: pointer;
   }
 
   .class-name {
-    font-size: 24px;
+    font-size: 1.5rem;
     font-weight: bold;
-    margin: auto 0.33333em;
+    margin: auto 5px;
   }
 
   .danger {
     color: $red-50;
-    margin-bottom: 0em;
+    margin-bottom: 0px;
```

Comment:
We don't need to use units when the value is 0.

Reply:
Updated!